src/main/resources/db/migration/V1_89__expense_updates.sql

```diff
@@ -0,0 +1,73 @@
+          sum(COALESCE(a.motorcycle_total_due, 0::numeric)) - sum(COALESCE(a.motorcycle_total_paid, 0::numeric)) as motorcycle_total_unpaid,
+          sum(COALESCE(a.car_total_due, 0::numeric)) AS car_total_due_total,
+          sum(COALESCE(a.car_total_paid, 0::numeric)) AS car_total_paid_total,
+          sum(COALESCE(a.car_total_due, 0::numeric)) - sum(COALESCE(a.car_total_paid, 0::numeric)) as car_total_unpaid,
+          sum(COALESCE(a.pedal_cycle_total_due, 0::numeric)) AS pedal_cycle_total_due_total,
+          sum(COALESCE(a.pedal_cycle_total_paid, 0::numeric)) AS pedal_cycle_total_paid_total,
+          sum(COALESCE(a.pedal_cycle_total_due, 0::numeric)) - sum(COALESCE(a.pedal_cycle_total_paid, 0::numeric)) as pedal_cycle_total_unpaid,
+          sum(COALESCE(a.parking_total_due, 0::numeric)) AS parking_total_due_total,
+          sum(COALESCE(a.parking_total_paid, 0::numeric)) AS parking_total_paid_total,
+          sum(COALESCE(a.parking_total_due, 0::numeric)) -  sum(COALESCE(a.parking_total_paid, 0::numeric)) as parking_total_unpaid,
+          sum(COALESCE(a.loss_of_earnings_due, 0::numeric)) AS loss_of_earnings_due_total,
+          sum(COALESCE(a.loss_of_earnings_paid, 0::numeric)) AS loss_of_earnings_paid_total,
+          sum(COALESCE(a.loss_of_earnings_due, 0::numeric)) -  sum(COALESCE(a.loss_of_earnings_paid, 0::numeric)) as loss_of_earnings_total_unpaid,
+          sum(COALESCE(a.childcare_total_due, 0::numeric)) AS childcare_total_due_total,
+          sum(COALESCE(a.childcare_total_paid, 0::numeric)) AS childcare_total_paid_total,
+          sum(COALESCE(a.childcare_total_due, 0::numeric)) - sum(COALESCE(a.childcare_total_paid, 0::numeric)) as childcare_total_unpaid,
+          sum(COALESCE(a.misc_total_due, 0::numeric)) AS misc_total_due_total,
+          sum(COALESCE(a.misc_total_paid, 0::numeric)) AS misc_total_paid_total,
+          sum(COALESCE(a.misc_total_due, 0::numeric)) -  sum(COALESCE(a.misc_total_paid, 0::numeric)) as misc_total_unpaid,
+          sum(COALESCE(a.subsistence_due, 0::numeric)) AS subsistence_due_total,
+          sum(COALESCE(a.subsistence_paid, 0::numeric)) AS subsistence_paid_total,
+          sum(COALESCE(a.subsistence_due, 0::numeric)) -  sum(COALESCE(a.subsistence_paid, 0::numeric)) as subsistence_total_unpaid,
+          sum(COALESCE(a.smart_card_due, 0::numeric)) AS smart_card_spend_total,
+          sum(
+                  CASE a.appearance_stage
+                      WHEN 'EXPENSE_ENTERED'::text THEN 1
+                      WHEN 'EXPENSE_EDITED'::text THEN 1
+                      ELSE 0
+                      END) AS pending_approval_count
+   FROM juror_mod.appearance a
+   GROUP BY a.juror_number, a.pool_number, a.loc_code;
+
+-- juror_mod.juror_expense_totals source
+
+CREATE OR REPLACE VIEW juror_mod.juror_expense_totals
+AS SELECT jes.juror_number,
+          jes.pool_number,
+          j.first_name,
+          j.last_name,
```

Comment:
have these intentionally been left here? assuming performance is Ok just dropping them from the previous query?

Reply:
We Dropped them from the sub-totals query and added them to the totals query  (As this makes the query run a little faster :))